app/assets/javascripts/application.js

```diff
@@ -19,185 +19,187 @@
+function getWeather() {
+  var enteredLocation = (locationField.value);
+  var url = `https://desolate-ocean-12145.herokuapp.com/api/v1/forecast?location=${enteredLocation}`;
 
+ fetch(url).then(function(response) {
+  return response.json();
+})
+.then(function(myJson) {
+  updateDOM(myJson);
+});
+}
 
-    //   var desc = data.weather[0].description;
-    //   var icon = data.weather[0].icon;
-    //
-      $('#location').html(location);
-      $('#current-temp').html(currentTemp);
-      $('#todays-high-temp').html("High: " + todaysHigh);
-      $('#todays-low-temp').html("Low: " + todaysLow);
-      $('#current-time').html(currentTime)
-      $('#current-icon').attr('src', weatherImages[currentIcon])
-      $('#details-icon').attr('src', weatherImages[detailsIcon])
-      $('#today-desc').html("Today: " + todayDesc)
-      $('#tonight-desc').html("Tonight: " + tonightDesc)
-      $('#feels-like').html("Feels Like: " + feelsLike)
-      $('#humidity').html("Humidity: " + humidity)
-      $('#visibility').html("Visibility: " + visibility)
-      $('#Uv').html("UV Index: " + uv)
-      $('#time-1').html(time1)
-      $('#time-2').html(time2)
-      $('#time-3').html(time3)
-      $('#time-4').html(time4)
-      $('#time-5').html(time5)
-      $('#time-6').html(time6)
-      $('#time-7').html(time7)
-      $('#time-8').html(time8)
-      $('#hour1-icon').attr('src', weatherImages[hour_1_Icon])
-      $('#hour2-icon').attr('src', weatherImages[hour_2_Icon])
-      $('#hour3-icon').attr('src', weatherImages[hour_3_Icon])
-      $('#hour4-icon').attr('src', weatherImages[hour_4_Icon])
-      $('#hour5-icon').attr('src', weatherImages[hour_5_Icon])
-      $('#hour6-icon').attr('src', weatherImages[hour_6_Icon])
-      $('#hour7-icon').attr('src', weatherImages[hour_7_Icon])
-      $('#hour8-icon').attr('src', weatherImages[hour_8_Icon])
-      $('#temp-1').html(temp1)
-      $('#temp-2').html(temp2)
-      $('#temp-3').html(temp3)
-      $('#temp-4').html(temp4)
-      $('#temp-5').html(temp5)
-      $('#temp-6').html(temp6)
-      $('#temp-7').html(temp7)
-      $('#temp-8').html(temp8)
-      $('#day-1').html(day1)
-      $('#day-2').html(day2)
-      $('#day-3').html(day3)
-      $('#day-4').html(day4)
-      $('#day-5').html(day5)
-      $('#icon-1').html(icon1)
-      $('#icon-2').html(icon2)
-      $('#icon-3').html(icon3)
-      $('#icon-4').html(icon4)
-      $('#icon-5').html(icon5)
-      $('#high-1').html("High: " + high1)
-      $('#high-2').html("High: " + high2)
-      $('#high-3').html("High: " + high3)
-      $('#high-4').html("High: " + high4)
-      $('#high-5').html("High: " + high5)
-      $('#day1-icon').attr('src', weatherImages[day_1_Icon])
-      $('#day2-icon').attr('src', weatherImages[day_2_Icon])
-      $('#day3-icon').attr('src', weatherImages[day_3_Icon])
-      $('#day4-icon').attr('src', weatherImages[day_4_Icon])
-      $('#day5-icon').attr('src', weatherImages[day_5_Icon])
+   // Update Dom
```

Comment:
everything looks good but I'm not a huge fan of this function, seems like it's doing a lot.  I might consider abstracting the weatherImages hash outside of this function